dhall.cabal

```diff
@@ -178,7 +178,9 @@ Library
         text                        >= 0.11.1.0 && < 1.3 ,
         transformers                >= 0.2.0.0  && < 0.6 ,
         unordered-containers        >= 0.1.3.0  && < 0.3 ,
-        vector                      >= 0.11.0.0 && < 0.13
+        vector                      >= 0.11.0.0 && < 0.13,
+        semigroups                  >= 0.8.3.1  && < 1 
```

Comment:
Actually, one more request: could you change this line to:

```
if !impl(ghc >= 8.0)
  build-depends: semigroups == 0.18.*
```

See: https://prime.haskell.org/wiki/Libraries/Proposals/SemigroupMonoid#RecommendedVariant